docs/troubleshooting.md

```diff
@@ -202,6 +202,28 @@ Example `hosts` file contents:
 
     10.0.2.2 my-local-website.com.docker.amazee.io
 
+### I get an error like `no such service: amazeeio-ssh-agent` when using `docker-compose up -d` even after pygmy started fine
+
+Restart pygmy and make sure that pygmy could start the ssh-agent container.
+
+    pygmy restart
+    pygmy status
+
+You could see a message like this:
+
+    [*] ssh-agent: Running as docker container amazeeio-ssh-agent, loaded keys:
+
+If you have recently updated Docker Desktop you should ensure that your turn off Docker Compose v2 support.
```

Comment:
```suggestion
#### Docker Desktop users

If you have recently updated Docker Desktop you should ensure that your turn off Docker Compose v2 support.
```

Reply:
This will make it obvious that the content is specifically for users of Docker Desktop.